app/models/post.rb

```diff
@@ -1,2 +1,3 @@
 class Post < ApplicationRecord
+  validates_presence_of :track, :artists, :image, :preview, :description
```

Comment:
maybe we users don't NEED to have a description??

Reply:
We agreed in the team that the purpose of the app is to interact through music, so we will keep the description as a validation!